app/views/events/_form.html.haml

```diff
@@ -0,0 +1,37 @@
+= simple_form_for [current_user, event] do |f|
+  = f.error_notification
+  = f.hidden_field :organizer_id, value: event.organizer.id
+  = f.input :name
+  = f.input :description, as: :text
+  = f.input :scheduled_at, as: :string
+  = f.simple_fields_for :event_address do |addr|
+    = addr.hidden_field :id
+    = addr.hidden_field :lat
+    = addr.hidden_field :lng
+    = addr.input :address
+  = f.input :scheduled_hours
+  = f.input :capacity, label: 'Maximum Count of Participants'
+  = f.input :is_private, as: :boolean
+  = f.button :submit, class: 'btn btn-success'
+
+= include_gmap_js
+
+:javascript
+  var addressPicker = new AddressPicker();
```

Comment:
`addressPicker` will be in the global scope for the page. Consider wrapping in an IIFE to reduce the scope and minimize what's in the global scope, and minimize the chance of name collisions.